### ISSUE TITLE
Le script ne fonctionne pas sur ARM64

### ISSUE THREAD

**Issue by melix** [metal3d/team-urgence#1]
Bonjour et merci pour ce script qui est vraiment super pratique !

J'ai essayé de le lancer sur une plateforme ARM64 (en l'occurrence, une VM Ubuntu 19.10 sur Freebox Delta).

Tout se passe bien jusqu'après le lancement des images docker, où le système plante avec cette erreur:

> bash: /usr/local/bin/docker-compose: cannot execute binary file: Exec format error

Je soupçonne, bien que je n'en ai pas la preuve, qu'il s'agit en fait de l'image docker téléchargée qui n'est pas compatible arm64.

Ca aurait été pratique, entre la delta et les Raspberry qui traînent :)

Merci !

**Comment by metal3d** [comment]
Tout à fait, je n'ai pas encore adapté à toutes les plateformes, je devrais le préciser sur le README.

Je me demande si il existe des builds de rocket et jitsi pour ARM cependant... Si oui, j'adapterai.

Merci beaucoup en tout cas pour le compliment ;)

**Comment by melix** [comment]
J'ai regardé rapidement et je ne trouve pas de version ARM. Du coup j'ai installé le couple Matrix+Riot.im , ça marche bien !
